google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/wrappers/AbstractMutation.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.wrappers;
```

Comment:
How necessary is it to put classes under `wrappers`? Generally, everything is put into the same package together in other clients.

Reply:
I think it's pretty important. When looking at the client I think having only the entry point at the top level makes it easier to navigate the source and makes it easier to discover tha api

Reply:
Well, you are introducing a different code structure than all of the other clients...

Can we think of a different name from `wrappers`? That doesn't convey anything meaningful to me; almost everything is a "wrapper" for something else.

Reply:
The generated clients don't have these classes because they expose the protobufs directly. These classes wrap the protobuf objects, which is why I named them wrappers. But you are right, the users don't really care that there are protobufs backing these classes. How would you feel about `models` or `dsl`?

Reply:
However, the rename will have to happen outside of this PR because that package already exists with classes:
https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/wrappers.

I opened an issue to come back to this:
https://github.com/GoogleCloudPlatform/google-cloud-java/issues/2916